packages/reporter/src/test/test.tsx

```diff
@@ -106,6 +106,7 @@ interface TestProps {
   runnablesStore: RunnablesStore
   scroller: Scroller
   model: TestModel
+  experimentalStudioEnabled: boolean
```

Comment:
I would have thought that as long as we set the top level prop in `<Reporter />` it'll get passed down - eg, since it's not optional, we don't need a default value? And TS **should** complain if we forgot to pass it down somewhere -- or maybe the TS config for `reporter` isn't strict enough?

I'll take a look but I don't think setting a default value for a non nullable boolean prop makes sense, does it?

Will definitely get a chance to re-assess once thing final PR opens, but it seems like we should only need default values for nullable props.